<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>1138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="file_list">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>371</width>
      <height>1001</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="file_edit">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>371</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>存放路径(请先设置好存放路径再添加录制窗口):</string>
    </property>
   </widget>
   <widget class="QListWidget" name="win_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>521</width>
      <height>1051</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="resizeMode">
     <enum>QListView::Adjust</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFunction">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFunction"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd"/>
   <addaction name="actionExit"/>
   <addaction name="actionHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/prefix1/image/add.png</normaloff>:/new/prefix1/image/add.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建录制窗口
Crtl+n</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/prefix1/image/exit.png</normaloff>:/new/prefix1/image/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出软件</string>
   </property>
   <property name="toolTip">
    <string>退出软件
Ctrl+j</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>帮助</string>
   </property>
   <property name="toolTip">
    <string>使用指南</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>win_list</tabstop>
  <tabstop>file_edit</tabstop>
  <tabstop>file_list</tabstop>
 </tabstops>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
